cmd/mdatagen/go.mod

```diff
@@ -100,3 +113,5 @@ replace go.opentelemetry.io/collector/internal/globalgates => ../../internal/glo
 replace go.opentelemetry.io/collector/consumer/consumerprofiles => ../../consumer/consumerprofiles
 
 replace go.opentelemetry.io/collector/consumer/consumertest => ../../consumer/consumertest
+
+replace github.com/atombender/go-jsonschema => github.com/ptodev/go-jsonschema v0.0.0-20240813163654-5518ba93ee84
```

Comment:
I had to [fork](https://github.com/ptodev/go-jsonschema/commits/otel-schema/) `go-jsonschema`, because it didn't contain some of the features which I needed. I hope we can upstream those.

Reply:
that would be best, especially if the maintainer of `go-jsonschema` is involved in this PR

Reply:
hey @ptodev now that the support for numeric comparison has landed, do you think we can look at the duration? so that would be one less blocker here ?